cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/asyncapi/add/ZillabaseAsyncapiAddCommand.java

```diff
@@ -85,25 +80,22 @@ protected void invoke()
 
     private String sendHttpRequest(
         InputStream content,
-        HttpClient client,
-        String artifactId)
+        HttpClient client)
     {
-        if (serverURL == null)
-        {
-            serverURL = ADMIN_SERVER_DEFAULT;
-        }
-
         String responseBody;
         try
         {
-            HttpRequest httpRequest = HttpRequest
-                .newBuilder(serverURL.resolve(ASYNCAPI_PATH))
+            HttpRequest.Builder httpRequest = HttpRequest
+                .newBuilder(ADMIN_SERVER_DEFAULT.resolve(ASYNCAPI_PATH))
                 .header("Content-Type", "application/vnd.aai.asyncapi+yaml")
-                .header("X-Registry-ArtifactId", "zillabase-asyncapi-%s".formatted(artifactId))
-                .POST(HttpRequest.BodyPublishers.ofByteArray(content.readAllBytes()))
-                .build();
+                .POST(HttpRequest.BodyPublishers.ofByteArray(content.readAllBytes()));
+
+            if (id != null)
+            {
+                httpRequest.header("X-Registry-ArtifactId", id);
+            }
```

Comment:
This apicurio-specific header should be added by the admin service not the command, to keep the OpenAPI spec for the admin service fully abstracted.

However, this is not blocking for the PR as we are using the commands at the moment, so please file a bug instead to track this.